ui/admin/app/components/form/host-set/aws/index.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormHostSetAwsComponent extends Component {
+  // =actions
+  @tracked showSyncIntervalSeconds = true;
+
+  @action
+  toggleSyncIntervalSeconds(model) {
+    this.showSyncIntervalSeconds = !this.showSyncIntervalSeconds;
+
+    if (!this.showSyncIntervalSeconds) {
+      model.sync_interval_seconds = null;
```

Comment:
Why are we setting the in the model `sync_interval_seconds` to `null` if this.showSyncIntervalSeconds does not exist?

Reply:
cause API expects us to send `null` as a default value when the checkbox is unselected.

Reply:
As per my understanding, if we are doing this to prepare data for the API, is the serializer who takes care of this.

That said, what I would expect is:
- Model defines that field.
- If the checkbox is unselected, then normalizer will take care of setting that value to `null`.
- If the checkbox is selected, then it will set a value.

Perhaps I am missing context, happy to jump on zoom call to discuss this.

Reply:
discussed offline: 
1. checkbox field is just for the UI toggling. There is no model state for this field
2. `model.sync_interval_seconds` is set to `null` on the component side when the user unselects the checkbox which clears the text box field and sends the expected `null` to the API.